adapters/webhdfs-adapter/src/test/java/com/connexta/replication/adapters/webhdfs/WebHdfsNodeAdapterTest.java

```diff
@@ -302,21 +302,27 @@ public void testQueryWithFailedItemIds() throws IOException {
 
     QueryResponse queryResponse = webHdfsNodeAdapter.query(queryRequest);
 
+    // expected sort order: failedFile2, failedFile1, file1, failedFile3
```

Comment:
Remove this comment or clarify the expected sort order, as the names have changed.